src/servlet/ServletName.java

```diff
@@ -0,0 +1,23 @@
+package servlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Map;
+
+public class ServletName extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException{
+        try(OutputStreamWriter writer=new OutputStreamWriter(resp.getOutputStream())){
+            Map<String,String[]> parameters = req.getParameterMap();
+            String name=parameters.getOrDefault("name", new String[]{"Kate"})[0];
+            if(req.getAttribute("name")!=null){
```

Comment:
You have probably copied these lines from our in-class exersise. There it played some (however not pretty) sense - to "override" a parameter map value with a falue ssigned in filter. But yoyu don't have no filter assigning values to requests around here. So the lines 17-19 are in vain here